app/views/posts/_post_presenter_show.html.erb

```diff
@@ -30,6 +30,17 @@
         <td scope="row">submitted at:</td>
         <td class="col2"><%= time_ago_in_words(post.created_at) %> ago</td>
       </tr>
+      <tr>
+        <td scope="row">vote count:</td>
```

Comment:
`vote count` is a little misleading here. the value being printed out is more of a `score`.

Reply:
true